doc-content/enterprise-only/cases/case-management-case-expression-proc.adoc

```diff
@@ -0,0 +1,37 @@
+[id='case-management-case-key-expression-proc-{context}']
+= Configuring case ID expressions
+
+The following procedures uses the *IT_Orders* sample project to demonstrate how set metadata attribute keys to customize expressions for generating the `caseId`.
+
+.Prerequisites
+* The *IT_Orders* sample project is open in {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects*. If there are existing projects, you can access the samples by clicking the *MySpace* default space and selecting *Try Samples* from the *Add Project* pull-down menu. If there are no existing projects, click *Try samples*.
+. Select *IT_Orders* and click *Ok*.
+. In the *Assets* window, click the *orderhardware* business process to open the designer.
+. Click on an empty space on the canvas and in the upper-right corner, click the *Properties* image:getting-started/diagram_properties.png[] icon.
+. Access the *Metadata Attributes* fields by expanding the *Advanced* menu.
+. Specify one of the following metadata attributes:
```

Comment:
The metadata attribute is "customCaseIdPrefix". LDAP, RPAD, TRUNCATE, and UPPER are "functions" that can be used as the value for the metadata attribute customCaseIdPrefix to create custom caseIds.

Reply:
Makes sense. Changed to:

. Specify one of the following functions for the `customCaseIdPrefix` metadata attribute:

* *LPAD* - Left padding
* *RPAD* - Right padding
* *TRUNCATE* - Truncate
* *UPPER* - Upper case

.Setting the UPPER function for the customCaseIdPrefix metadata attribute
image::cases/expressions.png[]

Reply:
I understand. Changed to:

. Specify one of the following functions for the `customCaseIdPrefix` metadata attribute:

* *LPAD* - Left padding
* *RPAD* - Right padding
* *TRUNCATE* - Truncate
* *UPPER* - Upper case

.Setting the UPPER function for the customCaseIdPrefix metadata attribute
image::cases/expressions.png[]